e2e/scenario_test/settings/not_set/basal_rates/settings.no_basal_rates.spec.js

```diff
@@ -0,0 +1,51 @@
+const { setup, pump, cgm, match, settings } = require('../../../../../src/index');
+
+describe('Pump Settings', () => {
+    beforeAll(async () => {
+        await setup.LaunchLoop();
+        await setup.LoadScenarios(device.id);
+        await cgm.Add();
```

Comment:
[nit] Consider, for clarify, `cgm.AddSimulator()`? Same for `pump.AddSimulator()`?